controller/src/main/java/io/pravega/controller/server/eventProcessor/ControllerEventProcessors.java

```diff
@@ -208,12 +211,7 @@ protected void shutDown() {
         }
     }
 
-    @Override
-    public boolean isReady() {
```

Comment:
Any specific reason why this is removed?

Reply:
isReady() is just invoking isRunning function. I have added a new instance variable `isReady` and a getter method. That is why removed this.